### ISSUE TITLE
Move `/lib/test` taglib here from core

### ISSUE THREAD

**Issue by jglick** [jenkinsci/junit-plugin#473]
I noticed a taglib in core sources https://github.com/jenkinsci/jenkins/blob/e7b46e917cef7273638a4c00cbfa88911c305a19/core/src/main/resources/lib/test/bar.jelly#L26 apparently forgotten during the `junit` plugin split. https://github.com/search?q=org%3Ajenkinsci+%22%2Flib%2Ftest%22&type=code shows it being used here from core https://github.com/jenkinsci/junit-plugin/blob/000693f1f36ea33a48a974d330f718cebf7044d3/src/main/resources/hudson/tasks/test/TestResult/index.jelly#L26-L34 yet it is also used in various other plugins like
* https://github.com/jenkinsci/maven-invoker-plugin/blob/58143debdef9bb58ea5c6c749b6944666d8b7e47/src/main/resources/org/jenkinsci/plugins/maveninvoker/MavenInvokerBuildAction/index.jelly#L30
* https://github.com/jenkinsci/multi-module-tests-publisher-plugin/blob/c613ef3241c454e18854b0a3abdde9b53b63167a/src/main/resources/com/cwctravel/hudson/plugins/multimoduletests/junit/ProjectResult/index.jelly#L32
* https://github.com/jenkinsci/tikal-multijob-plugin/blob/ca37a7c2778ef3598c651345c4dab82dd9c08445/src/main/resources/com/tikal/jenkins/plugins/multijob/MultiJobTestResults/index.jelly#L58
* https://github.com/jenkinsci/matrix-project-plugin/blob/06b7f47bc631d63212a680231b6979e4e61da7ee/src/main/resources/hudson/tasks/test/MatrixTestResult/index.jelly#L31
* https://github.com/jenkinsci/javatest-report-plugin/blob/5b703049afb8b6c7bbf970df8fd4936527356337/src/main/resources/hudson/plugins/javatest_report/TestCollection/index.jelly#L12
* https://github.com/jenkinsci/labeled-test-groups-publisher-plugin/blob/f2a89993ac8a4e0e5fd7c45ba5c313df12ebf8f4/src/main/resources/hudson/plugins/labeledgroupedtests/LabeledTestResultGroup/index.jelly#L32

as well as some taglib imports not apparently used like

* https://github.com/jenkinsci/tap-plugin/blob/431c6713253c95f2536e312b81ba698fe51913ae/src/main/resources/org/tap4j/plugin/model/TapTestResultResult/index.jelly#L26

Not to be confused with https://github.com/jenkinsci/junit-plugin/blob/000693f1f36ea33a48a974d330f718cebf7044d3/src/main/resources/lib/hudson/test which _was_ moved during the split.

**Comment by uhafner** [comment]
Yes, I noticed this as well while reviewing the styling changes in https://github.com/jenkinsci/jenkins/pull/6728. 

Rather than moving the tag I would suggest to restyle/rewrite the whole widget so that it would also reuse the current theme colors (green vs. blue).
